europace_security.yaml

```diff
@@ -68,6 +68,7 @@ flows:
 
       privatkredit:angebot:ermitteln: |
         ## Kreditsmartangebote ermitteln
+        Der Client ermöglicht die Ermittlung von Ratenkredit-Angeboten mit Schaufensterkonditionen.
```

Comment:
Hier bin ich jetzt unsicher ob das Ermitteln von Angeboten auf der Basis von Vorgangsdaten auch unter diese Beschreibung fällt? Daran arbeiten wir gerade, daher sollte es hier schon mit abgedeckt werden. Und auch die Formulierung mit dem "ermöglicht", weil es ja eigentlich ein Scope ist der dem Client ermöglicht etwas zu tun.

Reply:
> Angebote werden immer mit Vorgangsdaten ermittelt. 

Nein, dafür ist ja unsere Schaufenster-API da, da kann man auch ohne Vorgangsdaten ermitteln. Meine Frage geht in die Richtung, ob es dann missverstanden wird und angenommen wird, dass man nur die Schaufenster-API damit nutzen kann und nicht die API die wir aktuell entwickeln bei der man den Vorgang angeben kann. Beides soll möglich sein.

Reply:
Es gibt also bald zwei unterschiedliche APIs, auf die der Client mit der Berechtigung Kreditsmartangebote ermitteln zugreifen kann? Was kann die andere API, was die erste nicht kann? Bisher: Klick auf den Button Aktualisieren ermittelt Schaufensterkonditionen in einem Vorgang (auch wenn die Vorgangsdaten leer sind bis auf die gewünschte Darlehenssumme). Werden bei der neuen API Echtkonditionen ermittelt? Müssen die Vorgangsdaten vollständig ausgefüllt sein?

Reply:
Wie gesagt, die eine API kann Konditionen basierend auf Vorgangsdaten ermitteln (das ist genau das gleiche wie der Knopfdruck in KS) und die andere API kann Konditionen ohne Vorgangsdaten ermitteln (nur mit Laufzeit und Kreditwunsch). Aber in KS bezeichnen wir es ja nie als Schaufensterkonditionen, daher ist mir nicht klar ob das den Nutzer dann verwirrt, weil wir ja jetzt eine [API](https://github.com/europace/kex-angebote-api) haben, die explizit Schaufensterkonditionen liefert, auch wenn das hintenrum bei uns vielleicht alles das Gleiche ist.

Reply:
```suggestion
        Der Client kann Ratenkredit-Angebote und Schaufensterkonditionen ermitteln.
```

Reply:
Ich verstehe den Unterschied immer noch nicht. Ich sehe als Nutzer nur das Drücken auf Aktualisieren im Frontend. Und das kann Konditionen (unverbindlich, daher Schaufensterkonditionen) ohne Vorgangsdaten ermitteln. Es reichen Kreditwunsch und Laufzeit. "Der Client kann Ratenkredit-Angebote" ermitteln ist für mich das "Aktualisieren". Und das "mit" Schaufensterkonditionen. Nicht "und". Das "und" impliziert, dass es ermittelte unverbindliche Angebote gibt und noch etwas anderes, nämlich Schaufensterkonditionen. Was sollen diese dann noch sein?

Reply:
Es geht nicht um das Frontend, sondern um unsere public APIs, da haben wir jetzt die beiden Endpunkte https://github.com/europace/kex-angebote-api/#schaufensterkonditionen und https://github.com/europace/kex-angebote-api/#angebote und das möchte ich durch das Wording gern klar machen.

Reply:
Ich verstehe jetzt, dass ... api.europace.de/schaufenster z.B. von der VO-Homepage aus aufgerufen werden kann, um dort ohne einen KS-Vorgang Konditionen anzeigen zu können. Und ... api.europace.de/angebote mit einem KS-Vorgang aufgerufen werden kann, um Angebote zu ermitteln. Dann muss die Beschreibung oben reichen. Erst mit dem Zusatzwissen, dass es zwei APIs gibt, die der Client nutzen kann, und welche das sind, ergibt die Beschreibung Sinn. Besseres fällt mir jetzt auch nicht ein.